static/app/views/automations/components/actionFilters/eventAttribute.tsx

```diff
@@ -43,8 +39,7 @@ function AttributeField() {
 function MatchField() {
   const {condition, condition_id, onUpdate} = useDataConditionNodeContext();
   return (
-    <InlineSelectControl
-      styles={selectControlStyles}
+    <InlineSelectField
       name={`${condition_id}.comparison.match`}
       value={condition.comparison.match}
```

Comment:
Just noting that we have a few filters that use the same `match` or `age_comparison` datasets. Do you think it's worth DRYing these up?

Reply:
Unfortunately most of these expect different field names (ex. for `AGE_COMPARISON`, the ageComparisonHandler expects `comparison_type` while the latestAdoptedReleaseHandler expects `age_comparison`), so I think it's best to keep these separate in their own files for now